multiqc/modules/fgbio/groupreadsbyumi.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from fgbio GroupReadsByUmi
+"""
+
+############################################################
+######  LOOKING FOR AN EXAMPLE OF HOW MULTIQC WORKS?  ######
+############################################################
+#### Stop! This is one of the most complicated modules. ####
+#### Have a look at Kallisto for a simpler example.     ####
+############################################################
+
+from __future__ import print_function
+from collections import OrderedDict
+import io
+import json
+import logging
+import os
+import re
+import zipfile
```

Comment:
Most of these imports aren't used - please delete any that aren't needed.

Reply:
dependencies have been removed.